vllm/entrypoints/llm.py

```diff
@@ -42,10 +42,10 @@ class LLM:
             However, if the `torch_dtype` in the config is `float32`, we will
             use `float16` instead.
         quantization: The method used to quantize the model weights. Currently,
-            we support "awq", "gptq" and "squeezellm". If None, we first check
-            the `quantization_config` attribute in the model config file. If
-            that is None, we assume the model weights are not quantized and use
-            `dtype` to determine the data type of the weights.
+            we support "awq", "gptq", "fp8 and "squeezellm". If None, we first
```

Comment:
Let's add "(experimental)" for `fp8` here until we have better test coverage and have solved the performance problems :)